docs/en/_static/js/collapsed.js

```diff
@@ -1 +1,10 @@
 var collapsedSections = ['Migration Guides', 'API Reference']
+
+$(document).ready(function () {
+    $('.model-summary').DataTable({
+        "stateSave": false,
+        "lengthChange": false,
+        "pageLength": 20,
+        "order": []
+    });
+});
```

Comment:
You can put this section to `docs/en/_static/js/table.js`, and it has conflict with the existing setting on the `model-summary` class. May need to rename the class